src/Common/Hooks/useUrlFilters/useUrlFilters.ts

```diff
@@ -143,10 +145,31 @@ const useUrlFilters = <T = string, K = unknown>({
                 searchParams.delete(key)
             }
         })
+        localStorage.setItem(localStorageKey, JSON.stringify(parseSearchParams(searchParams)))
```

Comment:
leave a comment mentioning that we are skipping the primary filters like page number, sorting etc.